packages/mui-base/src/ModalUnstyled/ModalUnstyled.js

```diff
@@ -271,7 +272,7 @@ const ModalUnstyled = React.forwardRef(function ModalUnstyled(props, ref) {
             aria-hidden
             open={open}
             onClick={handleBackdropClick}
-            {...BackdropProps}
+            {...backdropProps}
```

Comment:
Doesn't have to be part of this pr, but looks like we are missing spreading `ownerState` here, no?

Reply:
Well spotted! I changed the code to use the same pattern as elsewhere (with `appendOwnerState`)